connectors/apache-connector/src/main/java/org/glassfish/jersey/apache/connector/ApacheConnector.java

```diff
@@ -648,7 +636,6 @@ private static InputStream getInputStream(final CloseableHttpResponse response)
             @Override
             public void close() throws IOException {
                 response.close();
-                super.close();
             }
```

Comment:
Another way to verify that my patch does NOT break connection reuse is running the code below - with my patch - while wiresharking traffic. Assuming the server honors Connection: Keep-alive, you'll see that both GET requests use the same TCP/HTTP connection.

    public static void main(String[] args) throws Exception
    {
      Client client = getClient();
       doJersey(client);
       doJersey(client);
     }
    
     public static void doJersey(Client client) throws Exception
     {
       String url = "http://...";
    
       WebTarget resource = client.target(url);
       Response response = resource
        .request()
        .get();
    
       response.readEntity(String.class);
       response.close();
     }